packages/website/ts/components/fill_order.tsx

```diff
@@ -61,6 +63,10 @@ interface FillOrderState {
 }
 
 export class FillOrder extends React.Component<FillOrderProps, FillOrderState> {
+    public static defaultProps: Partial<FillOrderProps> = {
+        isFullWidth: false,
+        shouldRenderHeader: true,
```

Comment:
nit: can this be `shouldHideHeader` just so we can have the default value of optional booleans be false?